index.html

```diff
@@ -0,0 +1,156 @@
+                    <div class="cont-description">Come get a talk from the most
+                        professional perspectives in the globe</div>
+                </div>
+                <div class="cont">
+                    <img src="images/exhibition.jpg" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Exhibition</h3>
+                    <div class="cont-description">Appreciate various art
+                        excellence on display organized from across
+                        continents</div>
+                </div>
+                <div class="cont">
+                    <img src="images/forum.jpg" alt="lecturer icon" width="70px"
+                        height="auto">
+                    <h3 class="cont-text">Forum</h3>
+                    <div class="cont-description">Come have the time to share
+                        your thoughts or questions with experts on
+                        whose answers you can rely </div>
+                </div>
+                <div class="cont">
+                    <img src="images/workshop.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Workshop</h3>
+                    <div class="cont-description">There will be a platformwhere
+                        you can build your own work whilst drawing
+                        inspiration from the live scenery</div>
+                </div>
+                <div class="cont">
+                    <img src="images/extra-badge-xxl.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Extras</h3>
+                    <div class="cont-description">There will be prizes to go
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
+
+            <nav class="wholepage">
+                <a href="" id="wholepage">SEE THE WHOLE PAGE</a>
+            </nav>
+
+        </article>
+
+
+        <section class="sec3">
+            <div class="guestscontainer">
+
+            </div>
+        </section>
+
+
+        <section class="sec4" id="sec4">
+            <div class="partnercontainer">
+                <h3 class="partner">Partners</h3>
+                <ul class="partneritems">
+                    <li class="partnerlist">Mozilla</li>
+                    <li class="partnerlist">Fine art America</li>
+                    <li class="partnerlist">Bona tour</li>
+                    <li class="partnerlist">Tama</li>
+                </ul>
+            </div>
+
+        </section>
+
+        <footer class="indexfooter">
+            <div class="footercontainer">&#169 AALA 2022</div>
+            <div class="footertext">2022 Annual Arts And Literary Appreciation</div>
+        </footer>
```

Comment:
Kindly fix the font-family for this section also 🤗

Reply:
Lato added thank you!!